packages/scripts/README.md

```diff
@@ -215,6 +215,10 @@ _Example:_
 This is how you execute those scripts using the presented setup:
 
 * `npm run test:e2e` - runs all unit tests.
+* `npm run test-e2e interactive` - runs all unit tests interactively.
```

Comment:
Shouldn't it look as follows? /cc @draganescu 
```md
* `npm run test-e2e -- --puppeteer-interactive` - runs all unit tests interactively.
* `npm run test-e2e FILE_NAME -- --puppeteer-interactive ` - runs one test file interactively.
* `npm run test-e2e:watch -- --puppeteer-interactive` - runs all tests interactively and watch for changes.
```

In watch mode, you rather don't pass the name of the file as by default it runs only for modified files.

Reply:
I made this change based on this PR's description: https://github.com/WordPress/gutenberg/pull/14129

Reply:
yes, but I didn't update the PR's description with the changes in the comments. @gziolo is right, in the end it was developed as:
```
* `npm run test-e2e -- --puppeteer-interactive` - runs all unit tests interactively.
* `npm run test-e2e FILE_NAME -- --puppeteer-interactive ` - runs one test file interactively.
* `npm run test-e2e:watch -- --puppeteer-interactive` - runs all tests interactively and watch for changes.
```